src/Tags/Vite.php

```diff
@@ -17,6 +17,13 @@ public function index()
             throw new \Exception('Please provide a source file.');
         }
 
-        return app(LaravelVite::class)($src);
+        $directory = $this->params->get('directory');
```

Comment:
Please add `build` as a second argument here, so it'll fall back to `build` if the parameter wasn't used.

Otherwise, if you don't use the param, it'll do `useBuildDirectory(null)` which changes the directory from `build` to the root.